evm_arithmetization/src/cpu/kernel/constants/global_metadata.rs

```diff
@@ -94,10 +94,23 @@ pub(crate) enum GlobalMetadata {
 
     KernelHash,
     KernelLen,
+
+    /// The address of the next available address in
+    /// Segment::AccountsLinkedList
+    AccountsLinkedListNextAvailable,
+    /// The address of the next available address in
+    /// Segment::StorageLinkedList
+    StorageLinkedListNextAvailable,
+    /// Length of the `AccountsLinkedList` segment after insertion of the
+    /// initial accounts.
+    InitialAccountsLinkedListLen,
+    /// Length of the `StorageLinkedList` segment after insertion of the
+    /// initial storage slots.
+    InitialStorageLinkedListLen,
 }
 
 impl GlobalMetadata {
-    pub(crate) const COUNT: usize = 47;
+    pub(crate) const COUNT: usize = 51;
```

Comment:
Maybe this magic number could be replaced with

```suggestion
    pub(crate) const COUNT: usize = Self::all().iter().map(|metadata| metadata::COUNT).sum();
    
```
which is less readable but also less magic.